tgui/packages/tgui/interfaces/CTFPanel.js

```diff
@@ -0,0 +1,64 @@
+import { useBackend } from '../backend';
+import { Box, Button, Section, Flex, Stack } from '../components';
+import { Window } from '../layouts';
+
+export const CTFPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const teams = data.teams || [];
+  const enabled = data.enabled || [];
+  return (
+    <Window
+      title="CTF Panel"
+      width={700}
+      height={600}>
+      <Window.Content scrollable>
+        <Section title={enabled} textAlign="center">
+          <Flex wrap="wrap" justify="space-around">
+            {teams.map(team => (
+              <Flex.Item key={team.name} minWidth="30%" mb={8}>
```

Comment:
I think instead of 30% this should be a fixed width that guarantees to contain all information about the team. And it should be something small. Then please adjust window size to result in a two column layout. I don't see why it should be three columns, there can only be an even number of teams.